### PR TITLE
[No Ticket] [Docs Only] Brew install method in README

### PR DESCRIPTION
**Pull request by meghfossa**
# Overview

This adds, `brew install --cask fossa` as viable install method for our users using macOs.

## Acceptance criteria

1. Users can use `brew` to install `fossa`

## Testing plan

We added automated test 2 months ago: https://github.com/fossas/fossa-cli/pull/1312 (no need for new CI tests)

1. Uninstall `fossa` from disk
2. Install `fossa` using `brew`
```sh
brew update # Fetch the newest version of Homebrew and all formulae
brew install --cask fossa
```

3. Perform `fossa --version`.
4. Perform `brew upgrade --cask fossa` (It should say latest is already installed)

## Risks
N/A

## Checklist

- [x] I added tests for this PR's change (or explained in the PR description why tests don't make sense).
- [x] If this PR introduced a user-visible change, I added documentation into `docs/`.
- [x] If this PR added docs, I added links as appropriate to the user manual's ToC in `docs/README.ms` and gave consideration to how discoverable or not my documentation is.
- [x] If this change is externally visible, I updated `Changelog.md`. If this PR did not mark a release, I added my changes into an `# Unreleased` section at the top.
- [x] If I made changes to `.fossa.yml` or `fossa-deps.{json.yml}`, I updated `docs/references/files/*.schema.json` AND I have updated example files used by `fossa init` command. You may also need to update these if you have added/removed new dependency type (e.g. `pip`) or analysis target type (e.g. `poetry`).
- [x] If I made changes to a subcommand's options, I updated `docs/references/subcommands/<subcommand>.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,14 @@ FOSSA CLI provides an install script that downloads the latest release from GitH
 ```bash
 curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | bash
 ```
+#### macOs with [brew](https://brew.sh/)
```
Comment (JeffreyHuynh1) [inline README.md:43]: should this be `macOS` ?

**Review by mostafakn — APPROVED** [review]
#1346

File: README.md [hunk 2]
```diff
@@ -40,6 +40,14 @@ FOSSA CLI provides an install script that downloads the latest release from GitH
...
 curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | bash
 ```
+#### macOS with [brew](https://brew.sh/)
+
+```sh
+brew update # Fetch the newest version of Homebrew and all formulae
+brew install --cask fossa
+```
+<!-- https://github.com/Homebrew/homebrew-cask/blob/master/.github/workflows/autobump.yml#L11 -->
+> The `fossa-cli` version in the brew cask is updated to the latest version every 3 hours.
```
Comment (JeffreyHuynh1) [inline README.md:50]:
```suggestion
> The `FOSSA CLI` version in the brew cask is updated to the latest version every 3 hours.
```

Change to proper name for CLI

**Comment by jssblck** [comment]
@meghfossa is there a reason this isn't merged yet?

**Comment by meghfossa** [comment]
> @meghfossa is there a reason this isn't merged yet?

No reason - it just felt sideway - I've set to auto merge now.

**Pull request merged**